lib/diagnostic.js

```diff
@@ -19,15 +19,15 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = true; // replace null with your response
 //
 
 // Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = [fire]; //a little confused by what 
```

Comment:
deathstar also has access because laserQuantity is defined within deathstar's scope